### PR TITLE
Add P-SEA

### PR DESCRIPTION
**Pull request by souravsingh**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by souravsingh** [comment]
xref https://github.com/biopython/biopython/pull/1336

**Comment by souravsingh** [comment]
Ping @peterjc

File: recipes/p-sea/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  build:
+    - gcc
+  run:
+
+test:
+  commands:
+    - p-sea -h
+
+about:
+  summary:  new efficient assignment of secondary structure from C_alpha.
```
Comment (peterjc) [inline recipes/p-sea/meta.yaml]: I think using "New" here is hardly true anymore, how about just "Efficient ..."?

File: recipes/p-sea/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package:
+  name: p-sea
+  version: "0.1" 
+
+source:
+  fn: P-SEA.tar.gz
+  url: ftp://ftp.lmcp.jussieu.fr/pub/sincris/software/protein/p-sea/P-SEA.tar.gz
+  md5: 405211399405ebd8784bdac329b42697
```
Comment (peterjc) [inline recipes/p-sea/meta.yaml]: Rather than an md5, can you provide the sha256 checksum instead please?

**Comment by peterjc** [comment]
TravisCI says it failed linting, due to ``missing_license``, unless you can find this information I suggest as a safe default using ``license: custom``

File: recipes/p-sea/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+build:
+  number: 0
+  skip: True # [osx]
```
Comment (peterjc) [inline recipes/p-sea/meta.yaml]: Don't skip Mac OS X - it should compile fine (at least, it does on my machine).

File: recipes/p-sea/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+package:
+  name: p-sea
+  version: "0.1"
```
Comment (peterjc) [inline recipes/p-sea/meta.yaml]: This is incorrect, it should be "0.99" as per:

```
$ ./p-sea -help | grep -i Version
| p-sea :   Version 0.99 LMCP       Sept   96   |
```

**Comment by souravsingh** [comment]
@peterjc The recipe is still getting errors.

**Comment by peterjc** [comment]
Not sure what is breaking here, from the Linux build:

```
Source cache directory is: /opt/conda/conda-bld/src_cache
Downloading source to cache: P-SEA.tar.gz
Downloading ftp://ftp.lmcp.jussieu.fr/pub/sincris/software/protein/p-sea/P-SEA.tar.gz
Success
Extracting download
Package: p-sea-0.99-0
source tree in: /opt/conda/conda-bld/p-sea_1504793230333/work
Traceback (most recent call last):
  File "/opt/conda/bin/conda-build", line 6, in <module>
    sys.exit(conda_build.cli.main_build.main())
  File "/opt/conda/lib/python3.5/site-packages/conda_build/cli/main_build.py", line 342, in main
    execute(sys.argv[1:])
  File "/opt/conda/lib/python3.5/site-packages/conda_build/cli/main_build.py", line 333, in execute
    noverify=args.no_verify)
  File "/opt/conda/lib/python3.5/site-packages/conda_build/api.py", line 97, in build
    need_source_download=need_source_download, config=config)
  File "/opt/conda/lib/python3.5/site-packages/conda_build/build.py", line 1522, in build_tree
    config=config)
  File "/opt/conda/lib/python3.5/site-packages/conda_build/build.py", line 1145, in build
    utils.check_call_env(cmd, env=env, cwd=src_dir)
  File "/opt/conda/lib/python3.5/site-packages/conda_build/utils.py", line 628, in check_call_env
    return _func_defaulting_env_to_os_environ(subprocess.check_call, *popenargs, **kwargs)
  File "/opt/conda/lib/python3.5/site-packages/conda_build/utils.py", line 624, in _func_defaulting_env_to_os_environ
    return func(_args, **kwargs)
  File "/opt/conda/lib/python3.5/subprocess.py", line 581, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/bin/bash', '-x', '-e', '/opt/conda/conda-bld/p-sea_1504793230333/work/conda_build.sh']' returned non-zero exit status 1
```

**Comment by souravsingh** [comment]
@bgruening I need some help in finishing the recipe.

File: recipes/p-sea/build.sh [hunk 5]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+#strictly use anaconda build environment
+CC=${PREFIX}/bin/gcc
+CXX=${PREFIX}/bin/g++
+
+./configure --prefix=$PREFIX
```
Comment (bgruening) [inline recipes/p-sea/build.sh]: The error message indicates that there is no `configure` script.
https://travis-ci.org/bioconda/bioconda-recipes/jobs/272909231#L827

**Comment by johanneskoester** [comment]
We  have made changes to our build system (migrated from Travis CI to  Circle CI). If you would like to continue working on this PR, please  reopen this PR and update with latest upstream/master.

**Pull request closed without merging**